src/index.html

```diff
@@ -7,8 +7,52 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+      type="text/css"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="navigationBar">
```

Comment:
Rather than using camel case for the classes you could use BEM methodology for naming, here it's optional however in later prs you will have to use it